docs/mesos/RpmCreateInstallMesos.md

```diff
@@ -0,0 +1,47 @@
+# Create and Install the Mesos + Net-Modules RPM
+This guide will walk you through building and installing Mesos + Net-Modules from RPMs. These RPMs will automate the installation of everything needed to run Mesos + Net-Modules in your cluster.
+
+## Build and Install the RPM
+The Mesos + Net-Modules RPMs can be built from the [net-modules repository](https://github.com/mesosphere/net-modules) by calling `make builder-rpm`. Once the RPMs are created, install the packages.
+> Note, Extra Packages for Enterprise Linux (EPEL) must be installed before installing Mesos + Net-Modules. You can download this package by calling `sudo yum install epel-release`
+
+    $ git clone https://github.com/mesosphere/net-modules.git
+    $ cd net-modules
+    $ git checkout integration/0.25
+    $ make builder-rpm
+    $ sudo yum install packages/rpms/RPMS/x86_64/*.rpm
+
+## Configure and Run Master
+
+We will be need to set the correct environment variables for the master. These environment variables are interpreted as command line arguments in the mesos-master application at runtime.
+> An explanation of the configuration options for Mesos can be found by running `mesos-init-wrapper -h`. 
+
+First, you will need set the ZooKeeper URL in `/etc/mesos/zk`. Modify the line to include the IP address of the host where ZooKeeper is running.
+
+The value in `/etc/mesos-master/quorum` may need to change depending on how many master hosts you have in your cluster. Mesos recommends that the quorum count is at least 1/2 the number of master hosts running. 
+
+Now you may run the mesos-master process on your master host.
+
+    $ sudo systemctl enable mesos-master.service
+    $ sudo systemctl start mesos-master.service
+
+## Configure and Run Agent(s)
+
+We will be need to set the correct environment variables for each agent. These environment variables are interpreted as command line arguments in the mesos-slave application at runtime.
+> An explanation of the configuration options for Mesos can be found by running `mesos-init-wrapper -h`. 
+
+Append the following lines to `/etc/default/mesos-slave` on each of your agent hosts. 
+
+    MESOS_RESOURCES="ports(*):[31000-31100]"
+    MESOS_MODULES=file:///calico/modules.json
```

Comment:
This file doesn't exist in mesos+netmodules, which I think is fine, so this would still be an appropriate place to set this environment variable. However if we do leave this here, we'll need to add a note that:

Mesos+netmodules RPM doesn't install calico/modules.json, you'll need to ensure that file is placed correctly. Follow one of the guides on  https://github.com/projectcalico/calico-docker/tree/master/docs/mesos#calico for instructions